web-app/js/portal/lang/en.js

```diff
@@ -179,7 +179,7 @@ OpenLayers.Lang.en = OpenLayers.Util.extend(OpenLayers.Lang.en, {
     resetActionText: 'reset',
 
     // FacetFilterPanel
-    addAnother: 'Add another filter',
+    addAnother: 'Add another filter - where possible',
```

Comment:
Is it really necessary to say 'where possible' as we now only show the icon when it *is* possible?